spark-connector/spark-common/src/test/java/org/apache/gravitino/spark/connector/integration/test/authorization/SparkAuthorizationIT.java

```diff
@@ -0,0 +1,260 @@
+    initTables();
+    initSparkEnv();
+  }
+
+  @AfterAll
+  @Override
+  public void stopIntegrationTest() throws IOException, InterruptedException {
+    containerSuite.close();
+    client.dropMetalake(METALAKE);
+    setEnv("HADOOP_USER_NAME", AuthConstants.ANONYMOUS_USER);
+    normalUserSparkSession.close();
+    super.stopIntegrationTest();
+  }
+
+  private void initMysqlContainer() throws SQLException {
+    containerSuite.startMySQLContainer(TestDatabaseName.MYSQL_CATALOG_MYSQL_IT);
+    mysqlUrl = containerSuite.getMySQLContainer().getJdbcUrl();
+    mysqlUsername = containerSuite.getMySQLContainer().getUsername();
+    mysqlPassword = containerSuite.getMySQLContainer().getPassword();
+    mysqlDriver = containerSuite.getMySQLContainer().getDriverClassName(MYSQL_CATALOG_MYSQL_IT);
+  }
+
+  private void initTables() {
+    client.createMetalake(METALAKE, "", new HashMap<>());
+    GravitinoMetalake gravitinoMetalake = client.loadMetalake(METALAKE);
+    gravitinoMetalake.addUser(NORMAL_USER);
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put(JdbcPropertiesConstants.GRAVITINO_JDBC_URL, mysqlUrl);
+    properties.put(JdbcPropertiesConstants.GRAVITINO_JDBC_USER, mysqlUsername);
+    properties.put(JdbcPropertiesConstants.GRAVITINO_JDBC_PASSWORD, mysqlPassword);
+    properties.put(JdbcPropertiesConstants.GRAVITINO_JDBC_DRIVER, mysqlDriver);
+    Catalog catalog =
+        client
+            .loadMetalake(METALAKE)
+            .createCatalog(
+                JDBC_CATALOG, Catalog.Type.RELATIONAL, "jdbc-mysql", "comment", properties);
+    SupportsSchemas schemas = catalog.asSchemas();
+    schemas.createSchema(JDBC_DATABASE, "", new HashMap<>());
+    SecurableObject securableObject =
+        SecurableObjects.ofCatalog(
+            JDBC_CATALOG,
+            ImmutableList.of(
+                Privileges.UseCatalog.allow(),
+                Privileges.UseSchema.allow(),
+                Privileges.SelectTable.allow(),
+                Privileges.CreateTable.allow()));
+    gravitinoMetalake.createRole(ROLE, new HashMap<>(), ImmutableList.of(securableObject));
+    gravitinoMetalake.grantRolesToUser(ImmutableList.of(ROLE), NORMAL_USER);
+  }
+
+  protected void putServiceAdmin() {
+    customConfigs.put(Configs.SERVICE_ADMINS.getKey(), ADMIN_USER);
+  }
+
+  private void initSparkEnv() {
+    SparkConf sparkConf =
+        new SparkConf()
+            .set("spark.plugins", GravitinoSparkPlugin.class.getName())
+            .set(GravitinoSparkConfig.GRAVITINO_URI, gravitinoUri)
+            .set(GravitinoSparkConfig.GRAVITINO_METALAKE, METALAKE)
+            .set("spark.sql.session.timeZone", TIME_ZONE_UTC);
+    setEnv("HADOOP_USER_NAME", NORMAL_USER);
+    normalUserSparkSession =
+        SparkSession.builder()
+            .master("local[1]")
+            .appName("Spark connector integration test")
+            .config(sparkConf)
+            .getOrCreate();
+  }
+
+  @Test
+  public void testTable() {
```

Comment:
Could you split the test to make the test purpose more clear?

Reply:
fixed